ecosystem/indexer/src/indexer/tailer.rs

```diff
@@ -15,6 +15,17 @@ use url::{ParseError, Url};
 
 diesel_migrations::embed_migrations!();
 
+pub fn remove_null_bytes_from_txn(txn: Arc<Transaction>) -> Transaction {
+    use serde_json::json;
+
+    let string_txn = json!(txn)
+        .to_string()
+        .replace("\\u0000", "")
```

Comment:
Can we ever see ‘\\u00’? I ask because should this be regex?
Alternatively for ‘\u{0000}’, is this the shortest form? Is it equiv to \{0}?